packages/server/src/message/firebase-relay.ts

```diff
@@ -27,31 +34,29 @@ function getMessagesRef() {
   return firebaseAppInsance.database().ref('messages');
 }
 
-async function postToHub(data = {}) {
-  const response = await fetch(`${process.env.SERVER_URL}/api/v1/channels`, {
-    method: 'POST',
-    headers: {
-      'Content-Type': 'application/json',
-    },
-    body: JSON.stringify(data),
-  });
-  return await response.json(); // parses response to JSON
-}
-
 function listenToFirebase() {
   const hubRef = getMessagesRef().child(HUB_ADDRESS.toLowerCase());
 
-  hubRef.on('child_added', snapshot => {
+  hubRef.on('child_added', async snapshot => {
     const key = snapshot.key;
     const value = snapshot.val();
     const queue = value.queue;
     if (queue === 'GAME_ENGINE') {
-      postToHub(value);
+      const outgoingMessage = await handleAppMessage(value);
```

Comment:
💡 throw an error here to be explicit about the fact that the current server does not support hub-play.